src/PlaneWaveBasis.jl

```diff
@@ -138,7 +141,8 @@ build_kpoints(basis::PlaneWaveBasis, kcoords) =
 @timing function PlaneWaveBasis(model::Model{T}, Ecut::Number,
                                 kcoords::AbstractVector, ksymops, symmetries=nothing;
                                 fft_size=nothing, variational=true,
-                                optimize_fft_size=false, supersampling=2) where {T <: Real}
+                                optimize_fft_size=false, supersampling=2,
+                                kgrid=nothing, kshift=nothing) where {T <: Real}
```

Comment:
this is pretty confusing that this constructor has both kgrid, kshift and kcoords, ksymops. But we probably need to fix this constructor someday ayway...

Reply:
Yes. I feel for the "K-Point grid and symmetry" we might actually need to introduce a datastructure. It's becoming out of hand all these k-arguments. And the meaning of the order of the k-Points inside and outside PlaneWaveBasis has diverged anyways since the MPI and spin stuff has been implemented.

Reply:
Yep, we have an issue for that: https://github.com/JuliaMolSim/DFTK.jl/issues/235

Reply:
Right I remember now.